chore: add building resources from `HjerlHede`

jdrueckert opened the pull request:
- the building resources should reside in a "more accessible" module
- thus they can be used in buildings outside of HjerlHede too without requiring to enable HjerlHede

jdrueckert committed:
chore: add building resources from `HjerlHede`

- the building resources should reside in a "more accessible" module
- thus they can be used in buildings outside of HjerlHede too without requiring to enable HjerlHede

skaldarnar commented on assets/blocks/brick-grey.block:
```diff
@@ -0,0 +1,7 @@
+    "displayName": "Grey Bricks",
+    "basedOn": "CoreAssets:Brick",
+    "tiles": {
+        "all": "hjerlhede:brick-grey"
```
```suggestion
        "all": "structuralResources:brick-grey"
```

I think this change is missing in all these block definitions.

skaldarnar reviewed (changes requested):
Also, you moved some `blockTiles/auto` to be non-auto tiles, but there is no accompanying `.block` definitions... need to double check that as well

jdrueckert committed:
fix: references to assets

jdrueckert committed:
fix: add block files for former auto assets

jdrueckert committed:
chore: update minor version

- this adds new features (hence > patch) that are backwards-compatible (hence < major) => minor
